src/services/catalog.service.ts

```diff
@@ -158,6 +160,7 @@ export class ImageItem implements IServiceItem {
     this.longDescription = this.getLongDescription();
     this.catsBySubCats = this.getCategoriesBySubCategories();
     this.isBuilder = _.includes(this.tags, 'builder');
```

Comment:
The latest tag references another tag. Tags can points to other tags in the same image stream or another image stream. We combine these in the version dropdown currently.